runtime/lang/menu_eo.utf-8.vim

```diff
@@ -75,8 +75,8 @@ menutrans Startup\ &Settings	                Agordoj\ de\ prav&aloroj
 menutrans &Global\ Settings			Mallo&kaj\ agordoj
 
 menutrans Toggle\ Pattern\ &Highlight<Tab>:set\ hls!	Baskuli\ emfazon\ de\ ŝa&blono<Tab>:set\ hls!
-menutrans Toggle\ &Ignore-case<Tab>:set\ ic!		Baskuli\ kongruon\ de\ uskle&co<Tab>:set\ ic!
-menutrans Toggle\ &Showmatch<Tab>:set\ sm!		Baskuli\ kongruon\ de\ kram&poj<Tab>:set\ sm!
+menutrans Toggle\ &Ignoring\ Case<Tab>:set\ ic!		Baskuli\ kongruon\ de\ uskle&co<Tab>:set\ ic!
+menutrans Toggle\ &Showing\ Matched\ Pairs<Tab>:set\ sm!		Baskuli\ kongruon\ de\ kram&poj<Tab>:set\ sm!
```

Comment:
I see that there was a mistake earlier here.
The verb "baskuli" is non-transitive according to both [ReVo](http://reta-vortaro.de/revo/dlg/index-2l.html#baskul.0i) and [PIV 2020](https://vortaro.net/#baskuli_kd), so it should have been "Baskuligi" in above 2 lines.